parquet-column/src/main/java/org/apache/parquet/column/values/boundedint/BitWriter.java

```diff
@@ -156,4 +162,15 @@ public int getCapacity() {
   public String memUsageString(String prefix) {
     return baos.memUsageString(prefix);
   }
+
+  public void close() {
+    currentByte = 0;
+    currentBytePosition = 0;
+    finished = false;
+    try {
+      baos.close();
+    } catch (IOException e) {
+      throw new ParquetRuntimeException("Error closing output stream.", e){};
```

Comment:
I have created an exception called OutputStreamCloseException that I generate in these locations instead of the anonymous innner classes I was using before. With a helper method like this though the exception name should probably be more generic. Would you prefer a more general exception with the helper method?